<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="54113" started-at="2021-07-06T22:35:52 IST" finished-at="2021-07-06T22:36:46 IST">
    <groups>
    </groups>
    <test name="Test1" duration-ms="8237" started-at="2021-07-06T22:35:52 IST" finished-at="2021-07-06T22:36:00 IST">
      <class name="testCases.HomePage">
        <test-method status="PASS" signature="initialize()[pri:0, instance:testCases.HomePage@27f981c6]" name="initialize" is-config="true" duration-ms="6960" started-at="2021-07-06T22:35:52 IST" finished-at="2021-07-06T22:35:59 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="baseHomePageNavigation()[pri:0, instance:testCases.HomePage@27f981c6]" name="baseHomePageNavigation" duration-ms="133" started-at="2021-07-06T22:35:59 IST" finished-at="2021-07-06T22:35:59 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- baseHomePageNavigation -->
        <test-method status="PASS" signature="endTesting()[pri:0, instance:testCases.HomePage@27f981c6]" name="endTesting" is-config="true" duration-ms="1083" started-at="2021-07-06T22:35:59 IST" finished-at="2021-07-06T22:36:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTesting -->
      </class> <!-- testCases.HomePage -->
    </test> <!-- Test1 -->
    <test name="Test2" duration-ms="9103" started-at="2021-07-06T22:36:00 IST" finished-at="2021-07-06T22:36:09 IST">
      <class name="testCases.LoginPage">
        <test-method status="PASS" signature="initialize()[pri:0, instance:testCases.LoginPage@51c668e3]" name="initialize" is-config="true" duration-ms="6204" started-at="2021-07-06T22:36:00 IST" finished-at="2021-07-06T22:36:07 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testCases.LoginPage@51c668e3]" name="login" duration-ms="1798" started-at="2021-07-06T22:36:07 IST" data-provider="getData" finished-at="2021-07-06T22:36:08 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[nidhi@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Password1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Valid user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="endTesting()[pri:0, instance:testCases.LoginPage@51c668e3]" name="endTesting" is-config="true" duration-ms="1059" started-at="2021-07-06T22:36:08 IST" finished-at="2021-07-06T22:36:09 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTesting -->
      </class> <!-- testCases.LoginPage -->
    </test> <!-- Test2 -->
    <test name="Test3" duration-ms="10386" started-at="2021-07-06T22:36:10 IST" finished-at="2021-07-06T22:36:20 IST">
      <class name="testCases.Courses">
        <test-method status="PASS" signature="initialize()[pri:0, instance:testCases.Courses@7b36aa0c]" name="initialize" is-config="true" duration-ms="6185" started-at="2021-07-06T22:36:10 IST" finished-at="2021-07-06T22:36:16 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="courses()[pri:0, instance:testCases.Courses@7b36aa0c]" name="courses" duration-ms="3646" started-at="2021-07-06T22:36:16 IST" finished-at="2021-07-06T22:36:19 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at testCases.Courses.courses(Courses.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- courses -->
        <test-method status="PASS" signature="endTesting()[pri:0, instance:testCases.Courses@7b36aa0c]" name="endTesting" is-config="true" duration-ms="170" started-at="2021-07-06T22:36:20 IST" finished-at="2021-07-06T22:36:20 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTesting -->
      </class> <!-- testCases.Courses -->
    </test> <!-- Test3 -->
    <test name="Test4" duration-ms="13525" started-at="2021-07-06T22:36:20 IST" finished-at="2021-07-06T22:36:34 IST">
      <class name="testCases.Contact">
        <test-method status="PASS" signature="initialize()[pri:0, instance:testCases.Contact@229c6181]" name="initialize" is-config="true" duration-ms="5873" started-at="2021-07-06T22:36:20 IST" finished-at="2021-07-06T22:36:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="contact()[pri:0, instance:testCases.Contact@229c6181]" name="contact" duration-ms="7521" started-at="2021-07-06T22:36:26 IST" finished-at="2021-07-06T22:36:33 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- contact -->
        <test-method status="PASS" signature="endTesting()[pri:0, instance:testCases.Contact@229c6181]" name="endTesting" is-config="true" duration-ms="122" started-at="2021-07-06T22:36:33 IST" finished-at="2021-07-06T22:36:34 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTesting -->
      </class> <!-- testCases.Contact -->
    </test> <!-- Test4 -->
    <test name="Test5" duration-ms="12089" started-at="2021-07-06T22:36:34 IST" finished-at="2021-07-06T22:36:46 IST">
      <class name="testCases.Practice">
        <test-method status="PASS" signature="initialize()[pri:0, instance:testCases.Practice@e350b40]" name="initialize" is-config="true" duration-ms="7220" started-at="2021-07-06T22:36:34 IST" finished-at="2021-07-06T22:36:41 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="practice()[pri:0, instance:testCases.Practice@e350b40]" name="practice" duration-ms="4732" started-at="2021-07-06T22:36:41 IST" finished-at="2021-07-06T22:36:46 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- practice -->
        <test-method status="PASS" signature="endTesting()[pri:0, instance:testCases.Practice@e350b40]" name="endTesting" is-config="true" duration-ms="132" started-at="2021-07-06T22:36:46 IST" finished-at="2021-07-06T22:36:46 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTesting -->
      </class> <!-- testCases.Practice -->
    </test> <!-- Test5 -->
  </suite> <!-- Suite -->
</testng-results>
